helpers/template.py

```diff
@@ -195,8 +193,12 @@ def _get_value(property_, true_value="", false_value="#",
             "NGINX_PUBLIC_PORT": config.get("exposed_nginx_docker_port",
                                             Config.DEFAULT_NGINX_PORT),
             "NGINX_EXPOSED_PORT": nginx_port,
-            "MAX_REQUESTS": config.get("max_requests", "512"),
-            "SOFT_LIMIT": int(config.get("soft_limit", "128")) * 1024 * 1024,
+            "UWSGI_WORKERS_MAX": config.get("uwsgi_workers_max"),
+            "UWSGI_WORKERS_START": config.get("uwsgi_workers_start"),
+            "UWSGI_MAX_REQUESTS": config.get("uwsgi_max_requests"),
+            "UWSGI_SOFT_LIMIT": int(config.get("uwsgi_soft_limit")) * 1024 * 1024,
+            "UWSGI_HARAKIRI": config.get("uwsgi_haraki"),
```

Comment:
okay, so it's already correct where it really matters :wink:
but let's be consistent with uWSGI and use `harakiri` everywhere